binary/Pos/Binary/Class/Primitive.hs

```diff
@@ -54,14 +59,26 @@ import           Pos.Binary.Class.Core (Bi (..), cborError, enforceSize, toCborE
 -- The output is represented as a lazy 'BSL.ByteString' and is constructed
 -- incrementally.
 serialize :: Bi a => a -> BSL.ByteString
-serialize = CBOR.Write.toLazyByteString . encode
+serialize = serializeWith 1024 4096
```

Comment:
Can you please document these two numbers somewhere? Just a single line. Also, L205 is `serialize`.